packages/react-component-library/src/components/TabSet/TabItem.tsx

```diff
@@ -6,50 +6,52 @@ import { StyledTab } from './partials/StyledTab'
 interface TabItemProps {
   tabId: string
   children: React.ReactElement | string
-  index: number
```

Comment:
I can't see any trace of this actually doing anything (as per the linter error), so I've removed it.